net/modemmanager/files/modemmanager.init

```diff
@@ -26,7 +33,12 @@ start_service() {
 
 	# Setup ModemManager service
 	procd_open_instance
-	procd_set_param command /usr/sbin/ModemManager
+	# If debugmode is enabled we start ModemManager in debug mode. This enables AT commands to modem.
```

Comment:
Instead of this debugmode, if you want AT command support via the DBus Command() interface, you can build MM using the `--with-at-command-via-dbus` configure option. This makes much more sense, if you know you're going to use those AT commands.

Reply:
Also, if you build using `--with-at-command-via-dbus` the log level is NOT changed to debug by default, so your concern about getting debug level when running using `--debug` no longer applies. A new MM build option in the openwrt setup makes more sense definitely.